### ISSUE TITLE
Add bash auto-completion scripts

### ISSUE THREAD

**Issue by fgiorgetti** [skupperproject/skupper#574]
This is related to: [#371](https://github.com/skupperproject/skupper/pull/371)

**Comment by agagliardi** [comment]
Is this feature already implemented? I tried to used `source <(./skupper completion)` but `bash-5.2$ ./skupper <tab>` fails with `_get_comp_words_by_ref: command not found` that seems something missing from Go Cobra

**Comment by agagliardi** [comment]
> @agagliardi, can you confirm you have the `bash-completion` package installed?
Yep.  `Package bash-completion-1:2.11-9.fc38.noarch is already installed`
